dot/telemetry/afg_received.go

```diff
@@ -19,17 +19,17 @@ var (
 )
 
 type afgReceived struct {
-	TargetHash   common.Hash `json:"target_hash"`
```

Comment:
can you confirm that substrate-telemetry is able to recognize these message (`afgReceived`, `AfgFinalizedBlocksUpTo`, basically any message type that you changed)

Reply:
@kishansagathiya I've set up 3 gossamer nodes and the substrate-telemetry server and here is the list of messages logged

````
message type Payload::SystemConnected
message type Payload::SystemInterval
message type Payload::PreparedBlockForProposing
message type Payload::NotifyFinalized
message type Payload::AfgFinalizedBlocksUpTo
message Payload::AfgReceivedPrevote
message Payload::BlockImport
message Payload::AfgReceivedPrecommit
message Payload::AfgReceivedCommit
````